app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -716,8 +715,6 @@ struct DeviceDelegateOculusVR::State {
     }
     if (renderMode == device::RenderMode::StandAlone) {
       vrapi_SetPropertyInt(&java, VRAPI_FOVEATION_LEVEL, standaloneFoveatedLevel);
-    } else {
-      vrapi_SetPropertyInt(&java, VRAPI_FOVEATION_LEVEL, immersiveFoveatedLevel);
```

Comment:
We should disable foveated here, otherwise both Gecko and FxR foveated can be applied together here (if foveated is enabled for standalone)

Reply:
Updated. I would like to make both of immersive and standalone mode use the same `standaloneFoveatedLevel` in OVR for now. I guess when we are ready doing it in GV::WebGL, we would disable foveated here and ask WebGL's framebuffer generate a foveated texture to us according to this `standaloneFoveatedLevel`.

Reply:
Ok, it makes sense. You have removed `vrapi_SetPropertyInt(&java, VRAPI_FOVEATION_LEVEL, standaloneFoveatedLevel);`